_posts/2020-09-06-how-the-system-tests-works.md

```diff
@@ -0,0 +1,607 @@
+---
+layout: post
+title:  "How the SYSTEM TESTS works"
+date: 2020-09-06
+author: maroš_orsák
+---
+
+Sometimes, we underestimate strength of testing. Many companies start verification of the software product belated. The process of testing is complex as many can not imagine. It can be highlighted by quote: "If we fail, we fall. If we succeed - then we will face the next task."
```

Comment:
You should start with something more optimistic and not how bad we are at system tests :-o